hw/xbox/nv2a/nv2a.h

```diff
@@ -314,8 +314,8 @@ typedef struct PGRAPHState {
 
 typedef struct CacheEntry {
     QSIMPLEQ_ENTRY(CacheEntry) entry;
-    unsigned int method : 14;
-    unsigned int subchannel : 3;
+    unsigned int method:14;
+    unsigned int subchannel:3;
```

Comment:
Is this how QEMU expects bitfields? I *strongly* dislike this. It makes it way harder to see what's going on:

```c
int pgraph_cache1;
int pgraph_cache:1;
```

I prefer the more obvious:

```c
int pgraph_cache1;
int pgraph_cache : 1;
```

Reply:
Yes, IIRC this is a suggestion by checkpatch. I'm indifferent here.